glusterd2/commands/volumes/volume-smartvol-txn.go

```diff
@@ -152,3 +160,59 @@ func txnUndoPrepareBricks(c transaction.TxnCtx) error {
 
 	return nil
 }
+
+func txnCleanBricks(c transaction.TxnCtx) error {
+	var volinfo volume.Volinfo
+	if err := c.Get("volinfo", &volinfo); err != nil {
+		c.Logger().WithError(err).WithField(
+			"key", "volinfo").Debug("Failed to get key from store")
+		return err
+	}
+
+	for _, b := range volinfo.GetLocalBricks() {
+		// UnMount the Brick if mounted
+		err := deviceutils.BrickUnmount(b.MountInfo.Mountdir)
+		if err != nil {
+			c.Logger().WithError(err).WithField("path", b.MountInfo.Mountdir).
```

Comment:
question:
if `BrickUnmount` fails, what we should do?
* `return back the error` 
*  `continue for loop`
* `do next steps`

Reply:
I will add a check to see mount exists. If mount exists and fails to unmount then return back the error.